src/modules/flow/iio/iio.json

```diff
@@ -822,6 +822,12 @@
              "name": "sampling_frequency"
          },
          {
+             "data_type": "int",
+             "default": -1,
+             "description": "Channel number. If -1, use the first channel",
+             "name": "option"
```

Comment:
Thanks @edersondisouza  for your comments. As the initial PR for the "multi-channel" support, I only tested the simple use-case(no buffer, no trigger, single node reading). Let me add the "two nodes on an app" in the next change